src/NzbDrone.Core.Test/ParserTests/LanguageParserFixture.cs

```diff
@@ -48,8 +48,17 @@ public class LanguageParserFixture : CoreTest
         [TestCase("Castle.2009.S01E14.HDTV.XviD.HUN-LOL", Language.Hungarian)]
         public void should_parse_language(string postTitle, Language language)
         {
-            var result = Parser.Parser.ParseTitle(postTitle);
-            result.Language.Should().Be(language);
+            var result = LanguageParser.ParseLanguage(postTitle);
+            result.Should().Be(language);
+        }
+
+        [TestCase("2 Broke Girls - S01E01 - Pilot.en.sub", Language.English)]
+        [TestCase("2 Broke Girls - S01E01 - Pilot.eng.sub", Language.English)]
+        [TestCase("2 Broke Girls - S01E01 - Pilot.sub", Language.Unknown)]
```

Comment:
FYI, .sub could be multiple sub streams. .idx file will tell. I do have a .idx .sub binary stream parser somewhere if it's required.